apps/storybook/.storybook/preview.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { Preview } from "@storybook/react";
+import "./global.css";
+import { TooltipProvider } from "@myds/react/tooltip";
+
+const preview: Preview = {
+  tags: ["autodocs"],
+  parameters: {
+    controls: {
+      matchers: {
+        color: /(background|color)$/i,
+        date: /Date$/i,
+      },
+    },
+  },
+  decorators: [
+    (Story) => (
+      <TooltipProvider>
+        {/* 👇 Decorators in Storybook also accept a function. Replace <Story/> with Story() to enable it  */}
+        <Story />
+      </TooltipProvider>
+    ),
+  ],
+};
```

Comment:
Please remove, bcs this will apply to all components. Refer to my other comment for soln